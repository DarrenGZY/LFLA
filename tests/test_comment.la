function main()
{
    var a = 1;
	# this is a comment a  = 2
	print(a);
	###
	this is a block comment;
	hope it works ;
	a = 32;
         ###
    print(a);
		
}
