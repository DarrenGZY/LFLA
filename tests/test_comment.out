#!/usr/bin/python
import numpy as np
from VecSpace import *
from InSpace import *
from AffSpace import *
def main() :
    a=1
    print(a)
    print(a)

main()
